Sage/carbon
# fix(focus-trap): update trap to use ref from content container FE-3795

fix #3770

### Proposed behaviour

Removes div wrapper from `FocusTrap` and replaces with fragment, now uses ref for the content container via `wrapperRef` prop

### Current behaviour

Animation does not work

### Checklist



### Additional context

Animation now working:

![2021-03-02 17 55 53](https://user-images.githubusercontent.com/44157880/109692628-a9d0dd00-7b80-11eb-8263-06bc79503ad2.gif)

### Testing instructions

`DialogFullScreen` story to test open animation
`Dialog`, `Sidebar` and `AdvancedColorPicker` stories will need a brush over to see that focus trap still functions as before
